### DISCUSSION TITLE
calendar to show reminders

### DISCUSSION THREAD

**Discussion by JTunX** [MDeLuise/plant-it#304] [Ideas]
would it be possible to get the calendar to show plant reminders so what is due to be done to a plant and have the ability to create custom reminders e.g. move plant inside/outside?

**Comment by MDeLuise** [comment]
Hi @JackTunny ,
Thank you for your suggestions, I'm happy to inform you that the reminder calendar feature has been completed and will be included in the next project release.

Regarding the ability to create custom reminders, I will definitely implement this as well. There is already an [issue](https://github.com/MDeLuise/plant-it/issues/197) tracking this feature.

Thanks again for your input!

<p align="center">
<img width="33%" src="https://github.com/user-attachments/assets/f477a879-8d85-4159-aee0-43e87f51dca7">
<img width="33%" src="https://github.com/user-attachments/assets/6b4bbafa-8a20-4115-9c53-5e19f899e45b">
</p>
